internal/cli/ops_manager_agents_list_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/fixtures"
+	"github.com/mongodb/mongocli/internal/mocks"
+)
+
+func TestOpsManagerAgentsList_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockAgentLister(ctrl)
+
+	defer ctrl.Finish()
+
+	expected := fixtures.Agents()
+
+	listOpts := OpsManagerAgentsListOpts{
+		globalOpts: &globalOpts{
+			projectID: "1",
+		},
+		store: mockStore,
+	}
+
+	t.Run("ProjectID is given", func(t *testing.T) {
```

Comment:
[nit] feel free to ignore this one but you don't need an extra scenario here as you only have one